### ISSUE TITLE
ggpiestats Cramer's V upper confidence intervals is always 1, and it shouldn't be

### ISSUE THREAD

**Issue by giladfeldman** [IndrajeetPatil/ggstatsplot#906]
in ggpiestats Cramer's V upper confidence intervals is always 1, and it shouldn't be. I'll explain and show an example from our code.

=

You're welcome to see our simulated data code to analyze chi-square on this Dropbox: https://www.dropbox.com/scl/fo/sz9gdascwnp2vo8diuw2d/h?rlkey=m6vupjllvwvjw4lmwbr9q7mbu&dl=0

The key rmarkdown file is:
Bown-et-al-2003-planned-data-analysis-simulated-data-V3.Rmd 
https://www.dropbox.com/scl/fi/qey063ic3hiz448a2suys/Bown-et-al-2003-planned-data-analysis-simulated-data-V3.Rmd?rlkey=ylgclzkeb7q6s721thr30kt52&dl=0
and the output is:
Bown-et-al-2003-planned-data-analysis-simulated-data-V3.html
https://www.dropbox.com/scl/fi/94f8no9eb668s67bq87br/Bown-et-al-2003-planned-data-analysis-simulated-data-V3.html?rlkey=jn1fy5jpih7lh129xnll2jqta&dl=0

You'll see we're doing a series of chi-squares, computing Cramer's V and then outputing a pie chart with ggpiestats.
However, the  

For example, the following code: 
```r
ggpiestats(
  data         = data,
  x            = bk_dv2_choiceR,
  y            = bk_condition,
  package      = "wesanderson",
  palette      = "Royal1",
  title        = "Study 1 Bank: Including lures",
  legend.title = "Choice of account"
)
```
which results in this piechart
![ggpiestats-cramer-v-CIU-issue-1](https://github.com/IndrajeetPatil/ggstatsplot/assets/22305310/952f1295-2d1c-4368-9fbe-481416ec4cda)

However the upper CIU for the Cramer's are wrong, if you run the following code:
```r
V_bkl <- cramersV(.bk_Table_l)
CI_bkl <- confIntV(.bk_Table_l)

data.frame(V = (V_bkl$output$cramersV), 
           Vlow = (CI_bkl$output$confIntV.fisher[1]), 
           Vhigh = (CI_bkl$output$confIntV.fisher[2]))
```
You'll get:
V   Vlow Vhigh
0.3338 0.2707 0.394

So, instead of 1 is should be 0.39.

Note: are the Bayesian stats in there look okay? I have no way of double checking that.

**Comment by IndrajeetPatil** [comment]
I think the difference is that _ggstatsplot_ defaults to _effectize_ default which is one-sided CIs (i.e. alternative hypothesis is not two-sided), and in such cases the upper bound for CI interval is fixed to `1.0`.
 
https://easystats.github.io/effectsize/reference/phi.html

**Comment by giladfeldman** [comment]
Thanks so much for the explanation, I appreciate that.
This is already helpful in helping us explain why our calculations defer from what's on the ggstatsplot output.

Wanted to follow up and ask...
This is a bit of an odd choice, no? I've personally rarely seen one-sided CIs reported, especially defaulting to 1, but maybe I'm not updated on these things.

Either way, is there already a way to change that somehow? if not - could you possibly add a quick way to change that to the two-sided option on _effectize_ so that we can keep that consistent with what we need for our project?
